src/index.html

```diff
@@ -15,8 +15,103 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="container">
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="/"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="navList">
+          <li class="link">
+            <a
+              href="#"
+              class="nav--link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
+              class="nav--link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
+              class="nav--link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li
+            class="link"
+            data-qa="hover"
+          >
+            <a
+              href="#"
+              class="nav--link"
+              data-qa="hover"
+            >
+              LAPTOPS &AMP; COMPUTERS
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
+              class="nav--link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
+              class="nav--link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
+              class="nav--link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
```

Comment:
All `href` attributes for the navigation links are currently set to '#'. Ensure these are updated to actual URLs or intended targets if this is not intentional.